src/lib/api/feedback.ts

```diff
@@ -0,0 +1,43 @@
+// Feedback is submitted to a Baserow table via the API
+
+const BASEROW_API_TOKEN = process.env.BASEROW_API_TOKEN;
+const BASEROW_FEEDBACK_TABLE_ID = process.env.BASEROW_FEEDBACK_TABLE;
```

Comment:
@eyeseast I believe this is our first integration with a 3rd party API as part of the SvelteKit rewrite. Open to alternative approaches to integration or managing secrets/endpoints.

Reply:
This is probably what we should be doing: https://kit.svelte.dev/docs/server-only-modules#private-environment-variables

That also means we need to handle the form server-side, but I think that's preferable anyway.

Reply:
I refactored the form action to run on the server, confirmed it's working on staging until it tries to call the Baserow API—it then fails because the env var isn't set.

Reply:
I can set those if you send them to me.